ser/infer.py

```diff
@@ -0,0 +1,36 @@
+import torch
+
+@torch.no_grad()
+def test_model_inference(params, model, image, label):
+    print(f"Runing inference on model - {params.name}")
+    print(f"Hyperparameters:\n",
+          f"EPOCHS: {params.epochs}\n",
+          f"LEARNING RATE: {params.learning_rate}\n",
+          f"Label of image: {label}"
+        )
```

Comment:
maybe a bit of a design choice, but arguably printing the hyperparams is more high-level logic. Whereas this inference function could just be the below inference logic.